### PR TITLE
Add Admin API to retrieve the status of EventProcessors.

### PR DESCRIPTION
**Pull request by saratry**

### CODE REVIEW

**Review by smcvb — APPROVED** [review]
Looks good to me :+1:

File: src/main/java/io/axoniq/axonserver/connector/admin/AdminChannel.java [hunk 1]
```diff
@@ -43,6 +44,21 @@
...
+     */
+    ResultStream<EventProcessor> eventProcessors();
+
+    /**
+     * Returns all the event processor registered by the specified application.
+     *
+     * @param application the application name
+     * @return the stream of all event processor registered to AxonServer by the specified application.
+     */
+    ResultStream<EventProcessor> eventProcessorsByApplication(String application);
```
Comment (abuijze) [inline src/main/java/io/axoniq/axonserver/connector/admin/AdminChannel.java:60]: I think we use the word "component" for this in other parts of the connector. Suggest: `ResultStream<EventProcessor> eventProcessorsByComponent(String componentName);`

File: src/main/java/io/axoniq/axonserver/connector/admin/impl/AdminChannelImpl.java [hunk 2]
```diff
@@ -69,6 +71,8 @@
 
     private static final int BUFFER_SIZE = 32;
     private static final int REFILL_BATCH = 8;
+    private static final Empty EMPTY = Empty.newBuilder().build();
```
Comment (abuijze) [inline src/main/java/io/axoniq/axonserver/connector/admin/impl/AdminChannelImpl.java:74]: `Empty.getDefaultInstance()` does the same.

File: src/main/java/io/axoniq/axonserver/connector/admin/impl/AdminChannelImpl.java [hunk 3]
```diff
@@ -85,6 +89,48 @@ public AdminChannelImpl(ClientIdentification clientIdentification,
...
+                // no app-level flow control available on this request
+                return null;
+            }
+
+            @Override
+            protected EventProcessor terminalMessage() {
+                return EventProcessor.newBuilder().build();
+            }
+        };
+        eventProcessorServiceStub.getEventProcessorsByApplication(request, results);
```
Comment (abuijze) [inline src/main/java/io/axoniq/axonserver/connector/admin/impl/AdminChannelImpl.java:130]: In other parts, we use the word "component". Suggest to rename to "getEventProcessorsByComponent".

**Review by abuijze — CHANGES_REQUESTED** [review]
Just some minor changes for consistent terminology

**Review by abuijze — APPROVED** [review]

**Pull request merged**